content/en/docs/setup/windows/user-guide-windows-nodes.md

```diff
@@ -157,16 +157,17 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
 In this section we'll cover configuring a Windows node from scratch to join a cluster on-prem. If your cluster is on a cloud you'll likely want to follow the cloud specific guides in the next section.
 
 #### Preparing a Windows Node
+
 {{< note >}}
-All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Admin).
+All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Admin). Windows Server 2019 (or later) is required. Windows 10 is not supported in v1.14.
```

Comment:
maybe again point to the supported OS section. i am asking to point to the section so that we have a single source of truth for supported OS instead of scattered all over the docs